test/models/validatable_test.rb

```diff
@@ -1,10 +1,9 @@
 # encoding: UTF-8
 # frozen_string_literal: true
-
-require 'test_helper'
+require "test_helper"
```

Comment:
```suggestion
require 'test_helper'
```

Prefer single-quoted strings when you don't need string interpolation or special symbols.
References: [rubocop](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/QuotedSymbols)